src/posts/2023-08-23.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Summer 2023 Fellow Reflections'
+author: 'Li Liu'
+image_url: '/images/posts/li-liu.jpg'
```

Comment:
**suggestion:** I noticed the image wasn't loading and I _think_ changing this hyphen to an underscore (to match the filename) should fix it.

```suggestion
image_url: '/images/posts/li_liu.jpg'
```